examples/src/Repositories/AccessTokenRepository.php

```diff
@@ -16,6 +16,20 @@
 
 class AccessTokenRepository implements AccessTokenRepositoryInterface
 {
+
+    /**
+     * @var string
+     */
+    private $issuer;
+
+    /**
+     * @param string $domain token issuer identifier
+     */
+    public function __construct($issuer)
```

Comment:
I know this is only an example but it might be worthwhile to make the issuer optional so it's backward compatible. Might make it easier to be accepted upstream

Reply:
I think it's better to show things done the right way in the examples..